app/validators/bytesize_validator.rb

```diff
@@ -7,14 +7,16 @@ class BytesizeValidator < ActiveModel::EachValidator
   CHECKS = { maximum: :<= }.freeze
   RESERVED_OPTIONS = %i[maximum too_long].freeze
 
-  ERROR_MISSING_OPTIONS_MESSAGE = "Specify the :maximum option.".freeze
-  ERROR_NON_NEGATIVE_MESSAGE = ":maximum must be a non-negative Integer".freeze
+  # seemingly unused value?
+  # ERROR_MISSING_OPTIONS_MESSAGE = I18n.t("validators.bytesize_validator.specify_the_maximum_option").freeze
```

Comment:
Refactor (blocking): Since this is unused and currently commented out, remove it since it's dead code.

```suggestion
```